### PR TITLE
feat(tokens): add new tokens

### PR DESCRIPTION
**Pull request by rzpcibot**
This PR was opened by the Token Upload GitHub action. It updates source token files based on the payload from Figma Plugin.

### CODE REVIEW

**Comment by kamleshchandnani** [comment]
@divyanshu013 tokens PR, you can take it froward from here

File: packages/blade/src/tokens/theme/paymentTheme.ts [hunk 1]
```diff
@@ -1009,7 +1033,7 @@ const colors: ColorsWithModes = {
           highContrast: globalColors.neutral.blueGrayDark[600],
         },
         level3: {
-          lowContrast: globalColors.neutral.blueGrayDark[800],
+          lowContrast: globalColors.neutral.blueGrayDark[900],
```
Comment (saurav12) [inline packages/blade/src/tokens/theme/paymentTheme.ts:1036]: Why is this pointing to `blueGrayDark`? Is this meant for the dark theme inside payments?

Comment (divyanshu013) [reply packages/blade/src/tokens/theme/paymentTheme.ts:1036]: The parent is `onDark` so I think it should be. It was always `blueGrayDark` but the shades are changing, wanted to confirm those.

**Comment by chaitanyadeorukhkar** [comment]
Closing this in favour of new tokens PR (Will be raised today)

**Pull request closed without merging**